frontend/__tests__/components/FilterPanel.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { render, screen } from '@testing-library/react'
+import FilterPanel from '../../src/components/FilterPanel'
+import '@testing-library/jest-dom'
+
+describe(FilterPanel, () => {
+  it('renders a preferences header', () => {
+    render(<FilterPanel />)
+    expect(screen.getByText('Your Preferences')).toBeInTheDocument()
+  })
+
+  it('renders a FilterCategory for each Dietary Restrictions filter', () => {
```

Comment:
This is me being nitpicky. 

For the test name, I'd recommend altering the test name to something along the lines of `displays a filter category for each...`. The current phrasing, particularly the use of `FilterCategory`, reveals implementation details by directly referencing the component name. Writing test descriptions that focus more on user experience and less on specific component names can help make the tests slightly decoupled from the component, and allows for more flexibility should we decide to rename it in the future.